File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/import-and-initialize/dart/import.dart
import 'package:absmartly_sdk/absmartly_sdk.dart';

void main() async {
  final sdk = Absmartly.create(
    endpoint: 'https://your-company.absmartly.io/v1',
    apiKey: 'YOUR-API-KEY',
    application: 'website',
    environment: 'development',
  );
}

File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/import-and-initialize/scala/import.scala
import com.absmartly.sdk.{SDK, SDKConfig}
import scala.concurrent.ExecutionContext.Implicits.global

val config = SDKConfig(
  endpoint = "https://your-company.absmartly.io/v1",
  apiKey = "YOUR-API-KEY",
  environment = "development",
  application = "website"
)

val sdk = new SDK(config)
